src/pages/developers/tbtc-przegląd-techniczny-systemu-2.pl.md

```diff
@@ -0,0 +1,74 @@
+
+Przykład: jeśli użytkownik zdeponuje 1,4 BTC w locie o wielkości 1 BTC, otrzyma on dowód pozwalający na wybicie dokładnie 1 TBTC (kwota odpowiadająca wielkości lota). W systemie pojawi się depozyt z nadpłatą, który inny użytkownik spodziewa się wykupić i wymienić 1 TBTC na 1,4 BTC. Użytkownik, który nadpłacił BTC będzie mógł, tak jak wszyscy inni użytkownicy, wymienić swój 1 TBTC na 1 BTC, ale dodatkowe 0,4 BTC zostanie utracone (chyba, że użytkownik zda sobie sprawę z popełnionego błędu i szybko wymieni swoje TBT na pierwotny depozyt 1,4 BTC).
+
+System zaakceptuje tylko UTXO większe niż wielkość lotu depozytu. Wszystkie inne BTC wysłane do grupy sygnatariuszy przepadają. Dlatego konieczne jest, aby deponenci wysyłali tylko jeden UTXO. Akceptowanie wielu UTXO od deponentów spowodowałoby znaczną złożoność łańcucha i opłaty za energie (ang. gas fees), ponieważ każde UTXO musiałoby zostać potwierdzone za pośrednictwem SPV i autoryzowanego podpisu. Sygnatariusze musieliby być zachęcani do podpisywania każdej transakcji, mimo, że całkowita wartość UTXO nie jest znana.
+
+## Random Beacon do wyboru sygnatariusza
+
+Siec Keep wymaga zaufanego źródła losowości do wybierania sygnatariuszy tBTC. W tym przypadku jest to tzw. BLS Threshold Relay.
+
+Gdy pojawi się żądanie utworzenia grupy sygnatariuszy, system tBTC użyje losowe ziarno (random seed) z bezpiecznego zdecentralizowanego random beacon’a do losowego wybrania członków grupy podpisujących z odpowiedniej puli sygnatariuszy. Sygnatariusze koordynują rozproszony protokół generowania kluczy, w wyniku czego powstaje publiczny klucz ECDSA, który jest używany do utworzenia adresu portfela, który jest następnie publikowany w łańcuchu hosta. Tak wygląda proces wyboru sygnatariusza.
+
+## Podpisy progowe
+
+tBTC wykorzystuje podpisy progowe (ang. threshold signatures) do generowania kluczy. Podpisy progowe pozwalają grupie sygnatariuszy na wygenerowanie pojedynczego klucza publicznego z zestawu „udziałów” klucza prywatnego. Ta metoda umożliwia podzbiorowi sygnatariuszy tworzenie podpisów w imieniu większej grupy. Użytkownicy mogą weryfikować grupy sygnatariuszy za pomocą jednego klucza publicznego odpowiadającego wielu kluczom prywatnym. Zapewnia to bezpieczeństwo bez spełnienia wymogów dla standardowych projektów multisig.
+
+Podpisy progowe zapewniają szereg korzyści:
+
+* Utworzenie grupy wymaga stosunkowo niewielkiej koordynacji Nie trzeba weryfikować zaufania względem członków grupy
+* Są odporne nawet wtedy, gdy połowa grupy działa na szkodę lub nie uda się jej wygenerować podpisu
+* W przypadku tBTC v1, grupy sygnatariuszy występują w modelu 3 z 3 – oznacza to, że są grupy 3 sygnatariuszy, które wymagają współpracy wszystkich 3 sygnatariuszy przy tworzeniu podpisów w imieniu grupy.
```

Comment:
"*oznacza to, że są grupy 3 sygnatariuszy, które wymagają współpracy wszystkich 3 sygnatariuszy przy tworzeniu podpisów w imieniu grupy.*"

proponuję uściślić na:

"*oznacza to, że gruby składają się z 3 sygnatariuszy i wymanagana jest współpraca ich wszystkich w celu stworzenia podpisu w imieniu grupy"*